### DISCUSSION TITLE
Could not find gem 'fastlane-plugin-match'

### DISCUSSION THREAD

**Discussion by chenguichun** [fastlane/fastlane#17462] [Q&A] [answered]
I kept reporting this error when I used the match function to pull the Apple development certificate from the git repository. Please help to solve it.

<img width="879" alt="企业微信截图_00ed7781-8c27-41eb-b373-a6c5814a9b93" src="https://user-images.githubusercontent.com/15317493/96390818-f4ad1200-11e8-11eb-9eaf-093bbc73f181.png">

<img width="947" alt="企业微信截图_aa0a6326-f517-408e-9aca-ea9e78e02d87" src="https://user-images.githubusercontent.com/15317493/96390815-f24ab800-11e8-11eb-89c3-644224a6daa1.png">

**Comment by ainame** [comment, marked as answer]
As the message says, there's no such gem called "fastlane-plugin-match". 
You need to install "fastlane" to use "match".

See this document.
https://docs.fastlane.tools/getting-started/ios/setup/#a-gemfile-macoslinuxwindows
